src/ast/helpers/stmt_create_table.rs

```diff
@@ -72,6 +72,7 @@ pub struct CreateTableBuilder {
     pub on_commit: Option<OnCommit>,
     pub on_cluster: Option<String>,
     pub order_by: Option<Vec<Ident>>,
+    pub big_query_config: Option<Box<BigQueryCreateTableConfiguration>>,
```

Comment:
Are there other dialects that will have similar variations on Create Table?
If yes, might be worth pulling this up to a more generic type - something like `CreateTableConfiguration` with variants for different dialects. 

Might be a premature abstraction.

Reply:
I think that makes sense to do! I've updated to use an enum as a result. There are a couple fields on the struct like the hive_* ones that could use that format so it seems reasonable to introduce yeah